# Prevents FLAC's build system from setting the compiler's optimization parameters (-O0, -O1, -O2, -O3, etc.) since Meson already handles that.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.12...3.31)
 
-if(NOT (CMAKE_BUILD_TYPE OR CMAKE_CONFIGURATION_TYPES OR DEFINED ENV{CFLAGS} OR DEFINED ENV{CXXFLAGS}))
-    set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo")
-endif()
+
+
+
 
 project(FLAC VERSION 1.5.0) # HOMEPAGE_URL "https://www.xiph.org/flac/")
 
